test/default-xpring-client-test.ts

```diff
@@ -32,6 +33,23 @@ const fakeErroringNetworkClient = new FakeNetworkClient(
   FakeNetworkClientResponses.defaultErrorResponses,
 )
 
+/**
+ * Convenience class for creating more specific Error objects to mimick grpc error responses.
+ */
+class FakeServiceError extends Error {
+  /**
+   * Construct a new FakeServiceError.
+   *
+   * @param message The text details of the error.
+   * @param code The grpc.status code.
+   */
+  constructor(public message: string, public code: grpc.status) {
```

Comment:
I think that lines 48/49 are redundant (they get automatically added by putting a `public` modifier in front of these properties.

Could you also make them `readonly`? It doesn't really matter, but immutability is a good habit.